ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -63,11 +64,11 @@ Verify RHODS User Groups
     ${auth_allowed}      Set Variable        ${AUTH_PAYLOAD[0]['spec']['allowedGroups']}
 
     ${rc}    ${adm_groups}=    Run And Return Rc And Output
-    ...    oc get configmap ${UPGRADE_CONFIG_MAP} -n ${UPGRADE_NS} -o jsonpath='{.data.adm_groups}'
+    ...    oc get configmap ${USERGROUPS_CONFIG_MAP} -n ${UPGRADE_NS} -o jsonpath='{.data.adm_groups}'
     Should Be Equal As Integers     ${rc}      0
 
     ${rc}    ${allwd_groups}=    Run And Return Rc And Output
-    ...    oc get configmap ${UPGRADE_CONFIG_MAP} -n ${UPGRADE_NS} -o jsonpath='{.data.allwd_groups}'
+    ...    oc get configmap ${USERGROUPS_CONFIG_MAP} -n ${UPGRADE_NS} -o jsonpath='{.data.allwd_groups}'
     Should Be Equal As Integers     ${rc}      0
 
     Should Be Equal    "${adm_groups}"    "${auth_admins}"   msg="Admin groups are not equal"
```

Comment:
Deletion has to be updated too:
https://github.com/red-hat-data-services/ods-ci/blob/a230cf6f5096e08e53d18913add8883b1653c8b4/ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot#L356-L363